stdlib/private/StdlibUnittest/RaceTest.swift

```diff
@@ -335,6 +337,36 @@ public func evaluateObservationsAllEqual<T : Equatable>(_ observations: [T])
   return .pass
 }
 
+// WebAssembly/WASI doesn't support multi-threading yet
```

Comment:
Can we not just mark the test as UNSUPPORTED on `OS=wasi` instead?

Reply:
This isn't a lit test. I don't think any tests in `StdlibUnittest` can be marked as `UNSUPPORTED`

Reply:
I guess compnerd said we can put unsupported statements for each test case, right?

Reply:
Sorry, not sure what you mean here, could you share an example?